operations/reservations.md

```diff
@@ -239,6 +241,22 @@ Returns all reservations specified by any identifier, customer or other filter.
 * `Connector`
 * `Navigator`
 
+#### Cancellation reason
+
+* `Other`
```

Comment:
Since we're now exposing this via API, I think we should do some consolidation because this is internal enum of quite inconsistent and sometimes even weird values. E.g. `Update` or `InvalidPayment`. We should not be exposing things via API that are not thought through much and only hacked together - that's how this enum happened. Noone put much thought in it, we've just kept adding things. That;s fine for internal use, but in open API, we should avoid this. CC @czech-man @patrikmiksovsky95

Reply:
@jirrick could you ? So we can release documentation on Monday/Tuesday ?

Reply:
I strongly vote not to expose this unless we do at least some basic consolidation and cleanup. We can still have it messy in the database and map it here to some cleaner enum.

Reply:
https://github.com/MewsSystems/mews/pull/21522/files its already open to somehow map it so we dont expose all of it

Reply:
Feel free to agree on what we dont want have there

Reply:
This is product decision and maybe needs a bit of discovery, so @czech-man pls.